src/advanced/bitcoin-anchoring.md

```diff
@@ -1,6 +1,6 @@
 # Anchoring Service
 
-<!-- cspell:ignore bitcoind,blockhash,lects,satoshis,txid,utxo,utxos -->
+<!-- cspell:ignore bitcoind,blockhash,satoshis,txid,utxo,utxos -->
 
 The anchoring service is developed to increase product security and
```

Comment:
I'd suggest to mention in this paragraph that the page describes the anchoring service for Exonum >=0.10 in order for the page to be more SEO-friendly. Also, it could make sense to link to the "older" page.